lib/daos/bookingDao.js

```diff
@@ -0,0 +1,76 @@
+import { bookings } from 'models';
+import { sequelize } from 'models';
+
+const attributes = ['id', 'fromLocation', 'toLocation', 'userId'];
+
+/**
+ * Creates a booking based on the data the user provides
+ * @param {object} data the pickup to destination provided by the user
+ * @returns the new created booking
+ */
+export const createOneBooking = async data => {
+    const { toLocation, fromLocation, userId } = data;
+
+    // The Lat and Lng will be sent as a stringified array
+    // for both to and from location, which can then be parsed
+    // and be used to reference and create points
+    const parsedToLocation = JSON.parse(toLocation);
+    const parsedFromLocation = JSON.parse(fromLocation);
+
+    const booking = await bookings.create({
+        fromLocation: sequelize.fn(
+            'ST_SRID',
+            sequelize.fn(
+                'ST_GeomFromText',
+                `POINT(${parsedFromLocation[0]} ${parsedFromLocation[1]})`
```

Comment:
Are we setting coordinates in the correct order [https://sequelize.org/api/v6/class/src/data-types.js~geometry](https://sequelize.org/api/v6/class/src/data-types.js~geometry)

GeoJson format: \[Longitude, Latitude\]